packages/design-system/src/components/Banner/Banner.md

```diff
@@ -0,0 +1,165 @@
+</ComponentHtmlMarkup>
+```
+Success
+```js
+<Banner size="medium" type="success" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+```
+```js noeditor
+<ComponentHtmlMarkup>
+<Banner size="medium" type="success" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+</ComponentHtmlMarkup>
+```
+### Large
+
+When there is enough space to fit the button on the side, then this component is transforming to Large form.
+
+Max text colum width is 720px
+
+Info
+```js
+<Banner size="large" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+```
+```js noeditor
+<ComponentHtmlMarkup>
+<Banner size="large" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+</ComponentHtmlMarkup>
+```
+
+Error / blocked
+```js
+<Banner size="large" type="error" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+```
+```js noeditor
+<ComponentHtmlMarkup>
+<Banner size="large" type="error" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+</ComponentHtmlMarkup>
+```
+Warning
+```js
+<Banner size="large" type="warning" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+```
+```js noeditor
+<ComponentHtmlMarkup>
+<Banner size="large" type="warning" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+</ComponentHtmlMarkup>
+```
+Success
+```js
+<Banner size="large" type="success" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+```
+```js noeditor
+<ComponentHtmlMarkup>
+<Banner size="large" type="success" onClose={()=>window.alert('close')}>A description with a maximum of 100 characters. That usually means only one or two sentences.</Banner>
+</ComponentHtmlMarkup>
+```
+
+### RWD support
+
+This component is not fully responsive. If you need an RWD support, you can use [Resize Observer](https://web.dev/en/resize-observer/) and apply appropriate styles. Here is an example of a large banner styled as a small one.
+
+```css
+.component {
+  position: relative;
+
+  .lc-banner--large {
+    max-width: 400px;
+
+    .lc-banner__content {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      width: 100%;
+
+      p {
+        margin: 14px 0 0 0;
+        max-width: 246px;
+      }
+    }
+  }
+}
```

Comment:
```suggestion
The component styles don't include RWD media queries. If you need an RWD support, you can use [Resize Observer](https://web.dev/en/resize-observer/) and change the size property of the component to the appropriate one. Also, you can write simple media queries with appropriate styles but remember that our styles can change in upcoming releases. Here is an example of a large banner styled as a small one.

```scss
@media screen and (max-width: 400px) {
  .your_component_class {
    position: relative;

    .lc-banner--large {
      max-width: 400px;

      .lc-banner__content {
        display: flex;
        flex-direction: column;
        justify-content: flex-start;
        width: 100%;

        p {
          margin: 14px 0 0 0;
          max-width: 246px;
        }
      }
    }
  }
}
```

Reply:
I meant something like this ☝️ No need to change styles when Resize Observer is used.

Reply:
+ i feel the example of css needs to be updated here because you added support for `children` and changed HTML structure

Reply:
done

Reply:
take a look at CSS example in my suggested change. I added a media query there. Could you do the same? 
Also there is not needed "an" in "If you need an RWD support" text. Please remove it to make it more English xD

Reply:
We need Ola. We need to improve our docs 🙈